pkgs/development/tools/web-ext/default.nix

```diff
@@ -16,6 +17,15 @@ buildNpmPackage rec {
 
   npmDepsHash = "sha256-KPBKUjCxva11w/E+Qhlx+1vikpCL7Hr9MiKenYHEVSU=";
 
+  preBuild = ''
+    export NODE_ENV=production
+  '';
```

Comment:
```suggestion
  env.NODE_ENV = "production";
```

Reply:
No, this breaks the build.

Reply:
Does `npmBuildFlags = [ "--production" ]` work?

Reply:
Yep, thanks! I should probably add you as co-author by now, do you want me to?